bin/common/install_tools.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset
+
+# Get version information and set destination dirs
+. "$(dirname "$0")/versions.sh"
+
+# Installs tools needed to build and run HCF
+SCF_BIN_DIR="${SCF_BIN_DIR:-/home/vagrant/bin}"
```

Comment:
Oh, this is a giant lie, and it's set in `versions.sh`? Then drop the lines here (and maybe add a comment if you want).